clang/lib/StaticAnalyzer/Checkers/BlockInCriticalSectionChecker.cpp

```diff
@@ -149,26 +149,34 @@ class BlockInCriticalSectionChecker : public Checker<check::PostCall> {
 private:
   const std::array<MutexDescriptor, 8> MutexDescriptors{
       MemberMutexDescriptor(
-          CallDescription(/*QualifiedName=*/{"std", "mutex", "lock"},
+          CallDescription(/*MatchAs=*/CDM::CXXMethod,
+                          /*QualifiedName=*/{"std", "mutex", "lock"},
                           /*RequiredArgs=*/0),
```

Comment:
Do we need the `CallDescription(...)` stuff? Couldn't we just use braces `{...}` like we usually do?

Reply:
I uploaded a commit that switches to `{...}`.

Interestingly the initializer of `std::array<...> BlockingFunctions` did not work with `{...}` instead of  `CallDescription(...)` -- but now that I looked at it I realized that it should be a `CallDescriptionSet` instead of a generic STL array.